docs-src/content/getting-started/streaming.md

```diff
@@ -2,13 +2,16 @@
 title: "Streaming Recognition"
 weight: 30
 ---
-The following example shows how to transcribe an audio file using Cubic's
-Streaming Recognize Request. It is assumed that the audio file contains raw
-samples, PCM16SLE like Cubic expects.  We will query the server for available
-models and use the first model to transcribe this speech.
+The following examples show how to transcribe audio streams using Cubic's
+Streaming Recognize Request. The stream can come from a file on disk or be
+directly from a microphone in real time. It is assumed that the audio stream 
+contains raw samples, PCM16SLE like Cubic expects. We will query the server
```

Comment:
Since we support more formats now, should we perhaps use WAV as the example, since it will be a more common use case ?